src/dotnet/commands/dotnet-msbuild/MSBuildForwardingApp.cs

```diff
@@ -31,7 +31,7 @@ public class MSBuildForwardingApp
             };
         
         private readonly IEnumerable<string> _msbuildRequiredParameters = 
-            new List<string> { "/m", "/v:m" };
+            new List<string> { "/m" };
```

Comment:
IMO it makes sense for build and run. I have mixed feelings for restore and others. @blackdwarf - please comment here.